src/App.js

```diff
@@ -21,14 +20,16 @@ class App extends Component {
         revenue: 0, 
```

Comment:
You don't _have_ to spell out every key here, but I'm sure it won't hurt. Just make sure you don't run into issues if the select movie has a piece of data missing. 

If you do really like to be specific about data types like this, you might want to look into Typescript!